evm.md

```diff
@@ -113,6 +113,7 @@ In the comments next to each cell, we've marked which component of the YellowPap
                       // ---------------
 
                       <activeAccounts> .Map </activeAccounts>
+                      <touchedAccounts> .Set </touchedAccounts>
                       <accounts>
```

Comment:
I think perhaps this is better grouped with other TX data (perhaps in the `substateLog`), as data that is only pertinent to an individual transaction.